.editconfig

```diff
@@ -0,0 +1,11 @@
+git # editorconfig.org
+
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+#indent_style = space
+#indent_size = 2
+trim_trailing_whitespace = true
+insert_final_newline = true
```

Comment:
I’m not sure what editor this file is for? I’m guessing it doesn’t affect anyone it’s not relevant to.

Reply:
Thanks for the good catches. I will address them. The .editconfig file tells text editors what settings to apply like how many spaces are in a tab. This helps make different people using different text editors comply to a team-agreed text-editing standard. We can discuss later on if it is needed (I assumed it is) and what needs to be in it if it is kept.

Reply:
@zoldello In your home folder you can make a global git ignore file for stuff like this. 

https://gist.github.com/subfuzion/db7f57fff2fb6998a16c

We don't want to check in personal configuration to the main repository.  In the global git ignore you should just add '.editconfig' so git won't check it in.